client/app/bundles/course/courses/components/CourseDisplay.tsx

```diff
@@ -0,0 +1,96 @@
+import { FC, useState } from 'react';
+import { Grid } from '@mui/material';
+import { CoursesEntity } from 'types/course/courses';
+import Pagination from 'lib/components/Pagination';
+import SearchBar from 'lib/components/SearchBar';
+import { injectIntl, defineMessages } from 'react-intl';
+import CourseInfoBox from './CourseInfoBox';
+
+interface Props {
+  intl?: any;
+  courses: CoursesEntity[];
+}
+
+const translations = defineMessages({
+  searchBarPlaceholder: {
+    id: 'search.placeholder',
+    defaultMessage: 'Search by course title',
+  },
+});
+
+const CourseDisplay: FC<Props> = (props) => {
+  const { intl, courses } = props;
+  // Ideally 24. Divisble by 2, 3 and 4. Also makes the pagination less awkward and causes less
```

Comment:
You can specifiy the exact numbers as tht types of itemsPerPage (eg 2 | 3 | 4 | 12 | 24) in the Pagination component .